taskcat/_amiupdater.py

```diff
@@ -336,7 +299,39 @@ def __init__(self, config, user_config_file=None, use_upstream_mappings=True):
         self.config = config
         self.boto3_cache = Boto3Cache()
         self.template_list = self._determine_templates()
-        self.regions = self._determine_testable_regions()
+        self.regions = self._get_regions()
+
+    def _get_regions(self):
+        profile = (
+            self.config.config.general.auth.get("default", "default")
```

Comment:
https://github.com/aws-quickstart/taskcat/blob/90d7b5c4e0b20fbd4211522e74ed3a44b41f42b8/taskcat/_common_utils.py#L215-L217

Was sourced in to prevent having to rewrite this every time we needed this logic. It was added after we had several papercuts in the `Config` class.

Reply:
but at this point I don't know what region I want, and can't just guess us-east-1, cause what if the default profile is not in the `aws` partition.